hackathon_site/dashboard/frontend/src/components/team/TeamCardAdmin/TeamCardAdmin.tsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import styles from "./TeamCardAdmin.module.scss";
+import Container from "@material-ui/core/Container";
+import Typography from "@material-ui/core/Typography";
+import Card from "@material-ui/core/Card";
+
+interface CardProps {
+    teamCode: string;
+    members: string[];
+}
+
+export const TeamCardAdmin = ({ teamCode, members }: CardProps) => {
+    const title = `Team ${teamCode}`;
+
+    return (
+        <Card style={{ minHeight: "200px" }}>
+            <Container className={styles.content_container}>
+                <Container className={styles.title}>
+                    <Typography variant={"h6"}>{title}</Typography>
+                </Container>
+                {members?.length
```

Comment:
since members is required, you can remove the ? after members

Reply:
done